kafka-client/src/main/java/dev/responsive/kafka/internal/db/LwtWriter.java

```diff
@@ -74,7 +93,7 @@ public CompletionStage<RemoteWriteResult> flush() {
       builder.setIdempotence(true);
       builder.addStatement(fencingStatementFactory.get());
 
-      for (int i = 0; i < batchSize && it.hasNext(); i++) {
+      for (int i = 0; i < maxBatchSize && it.hasNext(); i++) {
```

Comment:
Note that when the actual batch size is less than the `MAX_BATCH_SIZE`, the `it.hasNext` condition will catch the end of the batch anyways and so we don't need to pass in the batchSize as we were before